gateway/README.md

```diff
@@ -2,116 +2,167 @@
 
-```bash
-kubectl apply -f api-with-auth.yaml -n $KYMA_EXAMPLE_ENV
+Apply one of the `deployment.yaml` files from the `lambda` or `service` directory in this example.
+
+``` bash
+kubectl apply -f ./lambda/deployment.yaml -n {namespace}
+```
+
+#### Expose a service without authentication
+
+``` bash
+kubectl apply -f ./lambda/api-without-auth.yaml -n {namespace}
 ```
 
-Retrieve a token by accessing the [Dex Web application](https://dex-web.kyma.local/). Sign in with the admin@kyma.cx email address and the generic password from the [Dex ConfigMap](../../../resources/core/charts/dex/templates/pre-install-dex-config-map.yaml) file.
+#### Test the APIs without authentication
 
-Call the secured APIs:
+To perform a test, use the following command:
 
 ```bash
-# for the function:
-curl -i https://hello.kyma.local -H 'Authorization: Bearer <insert copied token here>'
-# for the service
-curl -ik https://http-db-service.kyma.local/orders -H 'Authorization: Bearer <insert copied token here>'
+curl -ik https://hello.kyma.local
+# > 200 Hello world
 ```
 
-### Cleanup
+#### Expose a service with authentication
 
-Run the following command to completely remove the example and all its resources from the cluster:
+There are two possible ways of exposing secured Api - using the default authentication settings, and using the custom settings. Authentication settings consist of the JWKS URI and the Issuer.
+
+``` bash
+# Create Api with the default authentication settings:
+kubectl apply -f ./lambda/api-with-default-auth.yaml -n {namespace}
+
+# OR
+
+# Create Api with the custom authentication settings:
+kubectl apply -f ./lambda/api-with-auth.yaml -n {namespace}
+```
+
+#### Test the APIs with authentication
 
 ```bash
-# for the function:
-kubectl delete function,api -l example=serverless-lambda -n $KYMA_EXAMPLE_ENV
+# To perform a test without token, use the following command:
+curl -ik https://<hostname>.kyma.local
+# > 401 Origin authentication failed.
 
-# for the service:
-kubectl delete service,deployment,api -l example=http-db-service -n $KYMA_EXAMPLE_ENV
+# To perform a test with token, use the following command:
+curl -ik https://<hostname>.kyma.local -H 'Authorization: <token>'
+# > 200 Hello world
 ```
 
-### Troubleshooting
+#### Cleanup
 
-- Exposing an API with authentication takes some time to work. Looking through the API resource can help. A good API should have:
+Remove the Api by using the `kubectl delete` command on the latest Api resource you have applied. For example, if you have created an Api from the `api-with-auth.yaml` file, run the following command:
 
-```yaml
-authenticationstatus
-  code: 2
+```bash
+kubectl delete -f ./lambda/api-with-auth.yaml -n {namespace}
 ```
 
+Remove a service or lambda by using the `kubectl delete` command` on the file from which the resource was created. See the example for lambda:
```

Comment:
``Remove a service or lambda by using the `kubectl delete` command on the file from which the resource was created. See the example for lambda:``